specs/command-viewer/index.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from "@sand4rt/experimental-ct-web";
+import { http, HttpResponse } from "msw";
+import { CommandViewer } from "../../src/command-viewer";
+import fs from "fs";
+import path from "path";
+import { fileURLToPath } from 'url';
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
```

Comment:
@tristangodfrey dit is volgens mij niet nodig wanneer de node polyfill goed ingeladen wordt.